cli/test/test_text2embeddings.py

```diff
@@ -38,23 +41,20 @@ def test_run_encoder_local(
                 Path(output_dir) / "test_pdf.json",
                 Path(output_dir) / "test_no_content_type.json",
             }
-            assert set(Path(output_dir).glob("*.npy")) == {
-                Path(output_dir) / "test_html.npy",
-                Path(output_dir) / "test_pdf.npy",
-                Path(output_dir) / "test_no_content_type.npy",
-            }
+            assert len(list(Path(output_dir).glob("*.npy"))) == 3 * len(SBERT_MODELS)
 
             for path in Path(output_dir).glob("*.json"):
                 assert ParserOutput.model_validate(json.loads(path.read_text()))
 
-            for path in Path(output_dir).glob("*.npy"):
-                assert np.load(str(path)).shape[1] == 768
+            # for path in Path(output_dir).glob("*.npy"):
+            #     assert np.load(str(path)).shape[1] == 768
 
-            # test_html has the `has_valid_text` flag set to false, so the numpy file
-            # should only contain a description embedding
-            assert np.load(str(Path(output_dir) / "test_html.npy")).shape == (1, 768)
+            # # test_html has the `has_valid_text` flag set to false, so the numpy file
+            # # should only contain a description embedding
+            # assert np.load(str(Path(output_dir) / "test_html.npy")).shape == (1, 768)
```

Comment:
These can be removed?

Reply:
leaving them for now as they might be useful debug steps later